### ISSUE TITLE
false positive for unused import

### ISSUE THREAD

**Issue by objectivePinta** [shyiko/ktlint#204]
I use the class for which I get the report like this:
`  val response = config.restTemplate.postForEntity(uri, excel, ThatClass::class.java)`

**Comment by shyiko** [comment]
```sh
$ cat A.kt
import x.y.z.ThatClass

fun main() {
    val response = config.restTemplate.postForEntity(uri, excel, ThatClass::class.java)
}
$ ktlint A.kt
// no errors printed
```

Please provide a complete example.

**Comment by JayNewstrom** [comment]
I've seen this happen as well. It was dealing with a val in one file, and using it in another package. I'll see if I can come up with a minimal sample to reproduce.

**Comment by JayNewstrom** [comment]
So it has to do with the naming of things.

```java
package jaynewstrom.minimal.another

import jaynewstrom.minimal.anotherThing

class Foo {
    val bar = anotherThing
}
```
And 
```java
package jaynewstrom.minimal

internal val anotherThing = ""
```

This minimal sample will reproduce it. Notice these are in different packages, and the name of the val has the name of the nested package in it.

**Comment by shyiko** [comment]
Thank you, Jay! I'll have a fix shortly.
